### PR TITLE
UnicodeNormalize: handle null values

### PR DESCRIPTION
**Pull request by eisber**

### CODE REVIEW

File: src/pipeline-stages/src/main/scala/UnicodeNormalize.scala [hunk 1]
```diff
@@ -49,10 +49,12 @@ class UnicodeNormalize(val uid: String) extends Transformer
...
     require(inputIndex != -1, s"Input column $getInputCol does not exist")
 
-    val normalizeFunc = (value: String) => Normalizer.normalize(value, Normalizer.Form.valueOf(getForm))
+    val normalizeFunc = (value: String) =>
+      if (value == null) null
+      else Normalizer.normalize(value, Normalizer.Form.valueOf(getForm))
 
     val f = if (getLower)
-      (value: String) => normalizeFunc(value.toLowerCase)
+      (value: String) => if (value == null) null else normalizeFunc(value.toLowerCase)
```
Comment (mhamilton723) [inline src/pipeline-stages/src/main/scala/UnicodeNormalize.scala]:
```suggestion
      (value: String) => Option(value).map(normalizeFunc(_.toLowerCase)).orNull 
```

Comment (eisber) [reply src/pipeline-stages/src/main/scala/UnicodeNormalize.scala]: for some reason the type cannot be resolve on _ . I only got it to work with

(value: String) => Option(value).map(s => normalizeFunc(s.toLowerCase)).orNull

Comment (mhamilton723) [reply src/pipeline-stages/src/main/scala/UnicodeNormalize.scala]: okay that works too if you dont mind changing it will remove some linter warnings

Comment (eisber) [reply src/pipeline-stages/src/main/scala/UnicodeNormalize.scala]: done

**Pull request merged**